master/pkg/model/defaults.go

```diff
@@ -31,12 +31,10 @@ const (
 
 // DefaultExperimentConfig returns a new default experiment config.
 func DefaultExperimentConfig(taskContainerDefaults *TaskContainerDefaultsConfig) ExperimentConfig {
```

Comment:
I could be wrong but I feel like this shouldn't be used anymore, I see it mostly just used by tests. cc @rb-determined-ai ?

Reply:
hmm this is not used any less than it was used before the pr, but after expconf pr, and If I'm not mistaken commands use it but only two parts of it so we can probably simplify it.

Reply:
used to impl db.Scanner interface for experimentconf and in a bunch of tests too so I'd keep it out of this pr but I can remove the reference from the commands config.